Utils/motion.hpp

```diff
@@ -1,7 +1,10 @@
-//#include "C:/Program Files (x86)/VEX Robotics/VEXcode Pro V5/sdk/vexv5/include/v5_color.h"
-#include "C:/Program Files (x86)/VEX Robotics/VEXcode Pro V5/sdk/vexv5/include/vex_units.h"
-#include "vex.h"
+#ifndef _motion_hpp_
+#define _motion_hpp_
+
+// #include "vex.h"  -- SHOULD NOT DEPEND ON A SPECIFIC PROJECT!!!
 #include <iostream>
+
+#define _USE_MATH_DEFINES
```

Comment:
What is this define, anyway?

Reply:
#define defines a macro variable.

Reply:
This is necessary to guarantee M_PI (and other math consts) are defined in cmath -- in MSVC. Not convinced it is needed for clang (cross compilation) as we do for VEX.

Some discussion is here: https://learn.microsoft.com/en-us/cpp/c-runtime-library/math-constants?view=msvc-170